system/jlib/jfile.cpp

```diff
@@ -2071,6 +2136,25 @@ CFileIO::CFileIO(IFile * _creator, HANDLE handle, IFOmode _openmode, IFSHmode _s
     extraFlags = static_cast<IFEflags>(extraFlags | expertEnableIFileFlagsMask);
     extraFlags = static_cast<IFEflags>(extraFlags & ~expertDisableIFileFlagsMask);
 
+    if (isContainerized() && (openmode!=IFOread)) // only containerized (with planes writing to storage types like blob for now)
+    {
+        const char *dataPath = "/var/lib/HPCCSystems/hpcc-data/";
+        if (startsWith(querySafeFilename(), dataPath))
+        {
+            DBGLOG("CFileIO: file '%s' is in data path '%s'", dataPath, querySafeFilename());
```

Comment:
Likely to create quite a bit of tracing (and line 2155)

Reply:
whoops, didn't mean to leave in. Will take out both lines.